src/Tizen.Multimedia/MediaTool/MediaFormatAudioMimeType.cs

```diff
@@ -92,6 +92,26 @@ public enum MediaFormatAudioMimeType
         /// </summary>
         WmaLossless = (MediaFormatType.Audio | MediaFormatDataType.Encoded | 0x10A3),
 
+        /// <summary>
+        /// AC3.
+        /// </summary>
+        Ac3 = (MediaFormatType.Audio | MediaFormatDataType.Encoded | 0x10B1),
```

Comment:
Please add the `<since_tizen>` tags with the current API level for all newly added enum values, because you are modifying enum, which already exists since `<since_tizen> 3</since_tizen>`.
```suggestion
        /// <since_tizen> 8 </since_tizen>
        Ac3 = (MediaFormatType.Audio | MediaFormatDataType.Encoded | 0x10B1),
```

Reply:
I fixed this.